cilium-cli/cli/connectivity.go

```diff
@@ -205,6 +205,7 @@ func newCmdConnectivityTest(hooks api.Hooks) *cobra.Command {
 	cmd.Flags().StringVar(&params.ConnDisruptTestXfrmErrorsPath, "conn-disrupt-test-xfrm-errors-path", "/tmp/cilium-conn-disrupt-xfrm-errors", "Conn disrupt test temporary result file (used internally)")
 	cmd.Flags().DurationVar(&params.ConnDisruptDispatchInterval, "conn-disrupt-dispatch-interval", 0, "TCP packet dispatch interval")
 
+	cmd.Flags().BoolVar(&params.IncludeNoUnexpectedPacketDropsTest, "include-no-unexpected-packet-drops-test", false, "Include no unexpected packet drops test")
```

Comment:
The double negation looks a bit weird to me `include-no-unexpected-packet-drops-test` set to `false` could mean that it is actually included so I would suggest doing the following instead:
```suggestion
	cmd.Flags().BoolVar(&params.IncludeUnexpectedPacketDropsTest, "include-unexpected-packet-drops-test", false, "Include unexpected packet drops test")
```
(it also matches the config option `include-conn-disrupt-test`)